user-news-and-weather-k8s/threeservices/user-news-and-weather/templates/weather-deployment.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: apps/v1beta2
+{{end}}
```

Comment:
This meant to be there?

Reply:
Seems like all deployment files are missing 3 lines. I'll submit a new merge request in a sec. thanks